qgis_plugin/tests/test_venv_manager.py

```
@@ -338,3 +338,244 @@ def fake_run(cmd, *args, **kwargs):
+
+    huge_stderr = "error: could not build wheel\n" + ("y" * 50000) + "\nCaused by: boom"
+
+    def fake_run_pip_install(**_kwargs):
+        return venv_manager._PipResult(1, "", huge_stderr)
+
+    monkeypatch.setattr(uv_manager, "uv_exists", lambda: False)
+    monkeypatch.setattr(venv_manager, "venv_exists", lambda _venv_dir=None: True)
+    monkeypatch.setattr(
+        venv_manager, "get_venv_python_path", lambda _venv_dir: "python"
+    )
+    monkeypatch.setattr(venv_manager, "_get_clean_env_for_venv", lambda: {})
```

Ruff (0.15.21)

[warning] Prefer `dict` over useless lambda

Replace with `lambda` with `dict`

(PIE807)